django/core/management/commands/inspectdb.py

```diff
@@ -126,6 +129,11 @@ def table2model(table_name):
 
                 yield ""
                 yield ""
+                if composite_key and type(composite_key) == list:
```

Comment:
I did it because currently the  Oracle backend sends the primary in string format, so this if block would take it too and give the message that multiple primary keys are present even if there is a single primary key. So that was the reason I did it since I wasn't able to find a way to return list of primary keys from Oracle. Its temporary though we can remove it once we find a way to return list of primary keys from Oracle.